src/js/service/startGame.js

```diff
@@ -0,0 +1,90 @@
+import RacingCar from '../model/RacingCar.js';
+import printSpinner from '../ui/printSpinner.js';
+import getRandomNumber from '../model/getRandomNumber.js';
+import { game } from '../constants.js';
+import renderCarStatus from '../ui/renderCarStatus.js';
+import hideSpinner from '../ui/hideSpinner.js';
+import { sortObjectByValue } from '../utils/index.js';
+
+const getWinnersName = (moveForwardCountArray) => {
+  const MOVE_FORWARD_CAR_NAME_INDEX = 0;
+  const MOVE_FORWARD_COUNT_INDEX = 1;
+
+  const firstWinner = moveForwardCountArray[0][MOVE_FORWARD_CAR_NAME_INDEX];
+  const winners = [firstWinner];
+
+  for (let i = 0; i < moveForwardCountArray.length - 1; i++) {
+    const currentWinner = moveForwardCountArray[i];
+    const nextWinner = moveForwardCountArray[i + 1];
+
+    if (
+      currentWinner[MOVE_FORWARD_COUNT_INDEX] > nextWinner[MOVE_FORWARD_COUNT_INDEX]
+    )
+      break;
+
+    winners.push(nextWinner[MOVE_FORWARD_CAR_NAME_INDEX]);
+  }
+
+  return winners;
+};
+
+export const getMoveForwardCount = (arr) => {
+  if (!Array.isArray(arr)) return;
+
+  return arr.reduce((acc, cur) => {
+    return cur >= game.THRESHOLD_SCORE ? acc + 1 : acc;
+  }, 0);
+};
+
+export const getWinners = (carName, record) => {
+  const moveForwardCount = {};
+
+  for (let i = 0; i < carName.length; i++) {
+    moveForwardCount[carName[i]] = getMoveForwardCount(record[carName[i]]);
+  }
+
+  const sortedMoveForwardCount = sortObjectByValue(moveForwardCount, 'desc');
+
+  return getWinnersName(sortedMoveForwardCount);
+};
```

Comment:
```suggestion

export const getWinners = (record) => {
  const carWithForward = Object.entries(([name, value]) => ({
    name,
    forwrad: getMoveForwardCount(value),
  }));
  
  const max = Math.max(...carWithForward.map(({ forward }) => forwrad));
  return carWithForward.filter(car => car.forwrad === max).map(car => car.name);
};
```

이렇게 표현할 수 있지 않을까요?
그러면 getWinnerNames도 불필요하겠네요.